modules/oadp-secrets-for-different-credentials.adoc

```diff
@@ -0,0 +1,123 @@
+
+.Procedure
+
+. Create a `credentials-velero` file with separate profiles for the backup and snapshot locations, as in the following example:
++
+[source,terminal]
+----
+[backupStorage]
+aws_access_key_id=<AWS_ACCESS_KEY_ID>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
+
+[volumeSnapshot]
+aws_access_key_id=<AWS_ACCESS_KEY_ID>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
+----
+
+. Create a `Secret` object with the `credentials-velero` file:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create secret generic {credentials} -n openshift-adp --from-file cloud=credentials-velero <1>
+----
+
+. Add the profiles to the `DataProtectionApplication` CR, as in the following example:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa-sample>
+  namespace: openshift-adp
+spec:
+...
+  backupLocations:
+    - name: default
+      velero:
+        provider: {provider}
+        default: true
+        objectStorage:
+          bucket: <bucket_name>
+          prefix: <prefix>
+        config:
+          region: us-east-1
+          profile: "backupStorage"
+        credential:
+          key: cloud
+          name: {credentials}
+  snapshotLocations:
+    - name: default
+      velero:
+        provider: {provider}
+        config:
+          region: us-west-2
+          profile: "volumeSnapshot"
+----
+endif::[]
+ifdef::installing-oadp-azure,installing-oadp-gcp,installing-oadp-ocs,installing-oadp-mcg[]
+If your backup and snapshot locations use different credentials, you create two `Secret` objects:
+
+* Backup location `Secret` with a custom name. The custom name is specified in the `spec.backupLocations` block of the `DataProtectionApplication` CR.
+* Snapshot location `Secret` with the default name, `{credentials}`. This `Secret` is not specified in the `DataProtectionApplication` CR.
+
+.Procedure
+
+. Create a `credentials-velero` file for the snapshot location in the appropriate format for your cloud provider.
+. Create a `Secret` for the snapshot location with the default name:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create secret generic {credentials} -n openshift-adp --from-file cloud=credentials-velero
+----
+
+. Create a `credentials-velero` file for the backup location in the appropriate format for your object storage.
+. Create a `Secret` for the backup location with a custom name:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create secret generic <custom_secret> -n openshift-adp --from-file cloud=credentials-velero
+----
+
+. Add the `Secret` with the custom name to the `DataProtectionApplication` CR, as in the following example:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa-sample>
+  namespace: openshift-adp
+spec:
+...
+  backupLocations:
+    - name: default
+      velero:
+        provider: aws
```

Comment:
@apinnick  I think we should use  `provider: {provider}` so it could shows the appropriate provider

Reply:
Good catch. Will fix.